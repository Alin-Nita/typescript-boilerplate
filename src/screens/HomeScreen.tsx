import Layout from "../components/Layout";

export const HomeScreen = () => {
  return (
    <Layout>
      <h1>Home</h1>
    </Layout>
  );
};

export default HomeScreen;
