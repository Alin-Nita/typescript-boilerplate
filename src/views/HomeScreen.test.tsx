import { render } from "@testing-library/react";
import HomeScreen from "./HomeScreen";

export default describe("HomeScreen", () => {
  it("Renders", () => {
    const { container } = render(<HomeScreen />);

    expect(container).toMatchSnapshot();
  });
});
