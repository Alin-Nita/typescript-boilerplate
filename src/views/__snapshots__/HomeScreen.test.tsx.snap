// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeScreen Renders 1`] = `
<div>
  <div
    class="layout"
  >
    <main
      class="layout__main"
    >
      <h1>
        Home
      </h1>
    </main>
  </div>
</div>
`;
